RELEASE/scripts/sl_ascend.ash

```diff
@@ -12703,8 +12704,57 @@ boolean L9_chasmBuild()
 		return true;
 	}
 
+
+	// -Combat is useless here since NC is triggered by killing Orcs...So we kill orcs better!
+	asdonBuff($effect[Driving Intimidatingly]);
+
+	// Check our Load out to see if spells are the best option for Orc-Thumping
+	if(setFlavour($element[cold]) && canUse($skill[Stuffed Mortar Shell]))
+	{
+		set_property("sl_useSpellsInOrcCamp", true);
+	}
+
+	if(setFlavour($element[cold]) && canUse($skill[Cannelloni Cannon]))
+	{
+	set_property("sl_useSpellsInOrcCamp", true);
+	}
+
+	if(canUse($skill[Saucegeyser]))
+	{
+		set_property("sl_useSpellsInOrcCamp", true);
+	}
+
+	if(canUse($skill[Saucecicle]))
+	{
+		set_property("sl_useSpellsInOrcCamp", true);
+	}
+
+	// Always Maximize and choose our default Non-Com First, in case we are wrong about the non-com we MAY have some gear still equipped to help us.
+	if(get_property("sl_useSpellsInOrcCamp").to_boolean())
+	{
+		print("Preparing to Blast Orcs with Cold Spells!", "blue");
+		addToMaximize("myst,20spell damage,40spell damage percent,20cold spell damage,-100 ml");
+		buffMaintain($effect[Carol of the Hells], 50, 1, 1);
+		buffMaintain($effect[Song of Sauce], 150, 1, 1);
+
+		// Since we are Buffing for Spells and Myst, set choice adventure just in case
+		set_property("choiceAdventure1345", 2);
+	}
+	else
+	{
+		print("Preparing to Ice-Punch Orcs!", "blue");
+		addToMaximize("muscle,50weapon damage,40weapon damage percent,20cold damage,-100 ml");
+		buffMaintain($effect[Carol of the Bulls], 50, 1, 1);
+		buffMaintain($effect[Song of The North], 150, 1, 1);
+
+		// Since we are Buffing for Weapons and Muscle, set choice adventure just in case
+		set_property("choiceAdventure1345", 1);
```

Comment:
I would rather the choice adventure be set to 0 for now, so that amisprediction won't be squashed.

Reply:
amisprediction?

Reply:
a misprediction

I want Mafia to abort if we aren't expecting that choice and receive it.

Reply:
I was thinking about this last night, and I hate random aborts because it flies in the face of automation.

So what I did instead, by default it WILL abort with a message alerting the user that it is currently buffed for option X. However I added an option to the "pre" section of the relay that allows them to let it choose the option for them.

Reply:
Oh, I agree that it shouldn't randomly abort forever! Sorry for the lack of clarity - I was imagining rolling this out under an `sl_beta` flag and leaving it in "fail if we mispredict" mode to check for any subtle bugs, but eventually (within a month, at most) fixing any mispredictions that show up and making it not abort on mispredictions.

I would rather have it always swallow mispredictions than than add a new preference for configuring it - otherwise we might have to re-organize that configuration page. :)

Reply:
Ok, I will remove the aggressive, and change it to the sl_beta property.

Reply:
It is changed. I also alerted the user that the abort is because Beta is Off.